src/test/java/org/jvnet/hudson/test/RealJenkinsRuleTest.java

```diff
@@ -389,4 +393,20 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         p.scheduleBuild2(0).waitForStart();
     }
 
+    @Test public void xStreamSerializable() throws Throwable {
+        rr.startJenkins();
+        // Neither ParametersDefinitionProperty nor ParametersAction could be passed directly.
+        // (In this case, ParameterDefinition and ParameterValue could have been used raw.
+        // But even List<ParameterValue> cannot be typed here, only e.g. ArrayList<ParameterValue>.)
+        var a = rr.runRemotely(RealJenkinsRuleTest::_xStreamSerializable, new XStreamSerializable<>(new ParametersDefinitionProperty(new StringParameterDefinition("X", "dflt"))));
```

Comment:
Maybe as static factory to avoid the syntax overhead.

```suggestion
        var a = rr.runRemotely(RealJenkinsRuleTest::_xStreamSerializable, XStreamSerializable.of(new ParametersDefinitionProperty(new StringParameterDefinition("X", "dflt"))));
```

Reply:
Had considered it. I guess it is slightly simpler that way…